test/integration/puppetlabs/services/jruby/jruby_puppet_pool_int_test.clj

```diff
@@ -508,7 +508,7 @@
            jruby-puppet (:jruby-puppet jruby-instance)]
        (try
          (testing "Various data types"
-           (is (= "0.0.0.0" (.getSetting jruby-puppet "bindaddress")))
+           (is (= "puppet" (.getSetting jruby-puppet "user")))
```

Comment:
Fair point. How about the ldapserver value? That shouldn't change in the near future.